OpenDreamRuntime/DreamManager.cs

```diff
@@ -41,6 +41,7 @@ public sealed partial class DreamManager {
     public Random Random { get; set; } = new();
     public Dictionary<string, List<DreamObject>> Tags { get; } = new();
     public DreamProc ImageConstructor, ImageFactoryProc;
+    public ushort ListPoolThreshold, ListPoolSize;
```

Comment:
unnecessarily small. Just use an int, for type consistency.
```suggestion
    public int ListPoolThreshold, ListPoolSize;
```